docs/charon/networking.md

```diff
@@ -1,6 +1,6 @@
 ---
 description: Networking
-sidebar_position: 7
+sidebar_position: 4
 ---
 
 # Charon networking
```

Comment:
It's annoying github won't let me comment on a line in this file you haven't touched, so commenting here but I want you to make a change to [line 10](https://github.com/ObolNetwork/obol-docs/pull/108/files#diff-3ee046445149c6b014eaf5d8bd3b134a89c28dc58132595322f184bef2565166L10) not this one pls. 

```suggestion
This document describes Charon's networking model which can be divided into two parts: the [*internal validator stack*](#internal-validator-stack) and the [*external p2p network*](external-p2p-network).
```